plugins/statistics.koplugin/calendarview.lua

```diff
@@ -523,25 +528,27 @@ function CalendarView:init()
                 })
             end,
         },
+        call_hold_input_on_tap = true,
         bordersize = 0,
-        margin = Screen:scaleBySize(20),
         text_font_face = "pgfont",
         text_font_bold = false,
     }
     self.page_info = HorizontalGroup:new{
         self.page_info_first_chev,
         self.page_info_spacer,
         self.page_info_left_chev,
+        self.page_info_spacer,
         self.page_info_text,
+        self.page_info_spacer,
         self.page_info_right_chev,
         self.page_info_spacer,
         self.page_info_last_chev,
     }
 
     local footer = BottomContainer:new{
         dimen = Geom:new{
-            w = self.content_width,
-            h = self.dimen.h,
+            w = self.inner_dimen.w,
+            h = self.inner_dimen.h,
```

Comment:
(And that's a BottomContainer, so its dimen.h should usually be the screen height.)